### PR TITLE
feat: Add model for changing password.

### PR DESCRIPTION
**Pull request by AsakusaRinne**
Fix: #21 

### What problem does this PR solve?
Add a model to support the password reset.

Related Issue: #21

### What is changed and how it works?
1. Add model for changing password.
2. Add corresponding sample code.
3. It seems there is a spelling mistake in the existing sample which write "built-in" as "build-in". I changed it so that the test can succeed. If it is in fact not a BUG, I will remove this change.

### CODE REVIEW

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@313843288 plz review

File: samples/ConsoleApp/Program.cs [hunk 1]
```diff
@@ -43,6 +43,23 @@
 
 var token = await client.RequestPasswordTokenAsync("admin", "123");
 ConsoleExtension.WriteLine("Get tokens by username and password...");
+if(token is null)
+{
+    Console.WriteLine("Failed to get the token.");
+    return;
+}
+client.SetBearerToken(token.AccessToken);
+Console.WriteLine($"token : {token.AccessToken}");
```
[the remark this answers is not in the document]
Comment (AsakusaRinne) [reply samples/ConsoleApp/Program.cs:52]: This is a sample of this sdk, or maybe you prefer unit test?

[the remark this answers is not in the document]
Comment (hsluoyz) [reply samples/ConsoleApp/Program.cs:52]: @AsakusaRinne OK, never mind. Finally we will move all example code out of this repo, into a dedicated example repo.

File: samples/ConsoleApp/Program.cs [hunk 2]
```diff
@@ -43,6 +43,23 @@
...
+
+var user = await client.GetUserAsync("admin");
+if(user is null)
```
[the remark this answers is not in the document]
Comment (AsakusaRinne) [reply samples/ConsoleApp/Program.cs:55]: Do you mean that I should add check for all nullable variables, I guess?

[the remark this answers is not in the document]
Comment (hsluoyz) [reply samples/ConsoleApp/Program.cs:55]: @AsakusaRinne `if(user is null)` -> `if (user is null)` , you can easily do all this by formating the code in the IDE

**Comment by hsluoyz** [comment]
@sagilio plz review

**Review by sagilio — CHANGES_REQUESTED** [review]
Others look good

File: src/Casdoor.Client/Models/CasdoorAccount.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Text.Json.Serialization;
+
+namespace Casdoor.Client.Models
+{
+    public class CasdoorAccount
```
Comment (sagilio) [inline src/Casdoor.Client/Models/CasdoorAccount.cs:19]: This class may be duplicated, and the name is not accurate.

**Review by sagilio — CHANGES_REQUESTED** [review]
Other changes look good.

File: src/Casdoor.Client/Models/CasdoorAccount.cs (hunk 3 above)
Comment (hsluoyz) [reply src/Casdoor.Client/Models/CasdoorAccount.cs:19]: @AsakusaRinne refer to how Go and Java SDKs name it

Comment (AsakusaRinne) [reply src/Casdoor.Client/Models/CasdoorAccount.cs:19]: > @AsakusaRinne refer to how Go and Java SDKs name it

In Go SDK it seems there hasn't been a method to reset password yet. In JAVA SDK it's named ```CasdoorAccountService```.

@sagilio Do you prefer to merge it into an existing class such as ```CasdoorUser```?

Comment (hsluoyz) [reply src/Casdoor.Client/Models/CasdoorAccount.cs:19]: @AsakusaRinne we can name it `CasdoorSdk`

Comment (sagilio) [reply src/Casdoor.Client/Models/CasdoorAccount.cs:19]: @hsluoyz @AsakusaRinne  Other SDKs do not have this model and it is not used in code, we can just delete it.

Comment (AsakusaRinne) [reply src/Casdoor.Client/Models/CasdoorAccount.cs]: The class has been removed

**Review by hsluoyz — APPROVED** [review]

**Review by sagilio — APPROVED** [review]
LGTM

**Pull request merged**